mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -62,7 +62,7 @@
 )
 
 _logger = logging.getLogger(__name__)
-_DOWNLOAD_CHUNK_SIZE = 10_000_000
+_DOWNLOAD_CHUNK_SIZE = 100_000_000  # 100 MB
```

Comment:
Use 100MB chunks, which is objectively faster due to the overhead of launching new processes